src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -960,4 +1054,52 @@ public void visitToken(DetailAST ast) {
             throw new IndexOutOfBoundsException("test");
         }
     }
+
+    private static class DummyCheck extends AbstractFileSetCheck {
```

Comment:
Rename to `DummyFileSet`.
I thought this was an `AbstractCheck` when reading it before looking at it's declaration.